translate/compiler.go

```diff
@@ -124,8 +128,45 @@ func compileStatement(s ast.Statement, bytecode *Bytecode) error {
 	}
 }
 
-// TODO: implement me w/ test cases :-)
-func compileAssignStatement(s *ast.AssignStatement, bytecode *Bytecode) error {
+// compileAssignStatement() compiles a assign statement.
+//
+// Ex)
+//
+// translate
+// 	'int a = 5'
+// to
+// 	'Push 5 Push <size of a> Push <offset of a> Mstore'
+//
+// stack will be
+//
+// 	[offset]
+// 	[size]
+// 	[value]
+//
+func compileAssignStatement(s *ast.AssignStatement, bytecode *Bytecode, memDefiner MemDefiner) error {
+	op, err := compileExpression(s.Value, bytecode)
+	if err != nil {
+		return err
+	}
+
+	memEntry := memDefiner.Define(s.Variable.Value, op)
+	if err != nil {
+		return err
+	}
+
+	size, err := encoding.EncodeOperand(memEntry.Size, encoding.EIGHT_PADDING)
```

Comment:
I think `memEntry.Size` should be `memDefiner.GetSizeOfEntry(s.Variable.Value)`. And `Size`, `Offset` (public) need to be changed to `size`, `offset`(private).

`memEntry.Size` is more clear? Or `memDefiner.GetSizeOfEntry(s.Variable.Value)` is more clear?